lib/features/auth/view/screens/sign_up_screen.dart

```diff
@@ -187,7 +195,7 @@ class SignUpScreenState extends State<SignUpScreen> {
                   iconSize: Sizes.iconSize,
                   radius: Sizes.circleButtonRadius,
                   formKey: formKey,
-                  handelSubmit: handelSubmit,
+                  handelSubmit: signUp,
```

Comment:
Man, you still didn't use the floatingActionButton of the Scaffold

Reply:
I will make this change when creating a new branch as the use of them was not exist in the branch that I was working with.